src/seedu/addressbook/Main.java

```diff
@@ -111,6 +112,9 @@ private CommandResult executeCommand(Command command)  {
             CommandResult result = command.execute();
             storage.save(addressBook);
             return result;
+        } catch (StorageOperationException e) {
+            ui.showToUser(Messages.MESSAGE_STORAGE_FILE_CANNOT_OPEN);
```

Comment:
In general, just showing the error message or printing the stack trace is not considered "good exception handling".
Think how you can improve the user experience in light of the exception that just occurred.